### ISSUE TITLE
A missing file is placed into the PLIST

### ISSUE THREAD

**Issue by yurivict** [pyscf/pyscf#934]
```
===> Checking for items in pkg-plist which are not in STAGEDIR
Error: Missing: %%PYTHON_SITELIBDIR%%/pyscf_lib_placeholder.so
===> Error: Plist issues found.
```

```pyrthon setup.py install --record plist``` produces the plist file with ```pyscf_lib_placeholder.so``` that is not installed.

**Comment by verena-neufeld** [comment]
Hi @yurivict , thanks for having raised this issue. Since this quite a few releases ago, has it been resolved?

**Comment by sunqm** [comment]
Fix in PR #1900
